libgadget/hydra.c

```diff
@@ -273,6 +303,8 @@ hydro_ngbiter(
             if(vsig > O->MaxSignalVel)
                 O->MaxSignalVel = vsig;
 
+            /* Note this uses the CurlVel of an inactive particle, which may not be
```

Comment:
DivVel as well? In principle we could have drifted them to current time since the acceleration is known? Did you check if we drift these quantities at all?

Reply:
Yes,  DivVel as well, I think. As far as I can see we don't drift them. I don't know that we can easily drift them because they depend on the relative velocities between the particle and neighbours. It might be worthwhile implementing a timestep limiter as a fix.